### ISSUE TITLE
Rule Request: delete commented-out code

### ISSUE THREAD

**Issue by yonat** [realm/SwiftLint#2579]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### New rule request

> 1. Why should this rule be added?

Sometimes we comment-out code while programming, and then forget to uncomment or delete that code.

Commented-out code is either a bug and should be un-commented, or cruft to be deleted.

> 2. Provide several examples of what _would_ and _wouldn't_ trigger violations.

Triggering:

```swift
//        descriptionLabel.font = UIFont.openSans(16.0)
//        descriptionLabel.numberOfLines = 4
//        descriptionLabel.lineBreakMode = .byWordWrapping
```

Non-triggering:

```swift
// words that are not code

    // MARK: something

    /// documentation
```

> 3. Should the rule be configurable, if so what parameters should be configurable?

Parameter: the number of lines to consider a violation.
Maybe allow 1-2 lines of commented-out code, but definitely not 10.

> 4. Should the rule be opt-in or enabled by default? Why?

Enabled by default, commented-out code is a potential bug.
